tests/unit/index.test.js

```diff
@@ -64,6 +64,25 @@ Tape('start with false filter with output to stdout', (t) => {
   })
 })
 
+Tape('start with false filter with output to stdout with non-JSON', (t) => {
+  beforeEach()
+  t.plan(2)
+  const stdin = StdInMock.stdin()
+  StdOutMock.use({ stdout: true, stderr: true })
+  const { start } = require(requirePath)
+  const config = { filter () { return false }, output: { path: 'test.log', options: {}, notjson: true }, exit: false }
+  start(config)
+  stdin.send(Buffer.from('{}', 'utf8'))
```

Comment:
Please include a test that shows what happens when _non-JSON_ text is sent to stdin.